### PR TITLE
feat(alert): add ability to put a button within an alert

### LINKED ISSUE #2848: ClayAlert with action button
In [clay issue 2765](https://github.com/liferay/clay/issues/2765), @matuzalemsteles mentioned the need to add some extra markup to align the button with the text and add extra spacing in alerts. As we are working on [LEXI-780](https://issues.liferay.com/browse/LEXI-780), we would like to know if this is already implemented?

### PR DESCRIPTION
**Pull request by bryceosterhaus**
fixes #2848

### CODE REVIEW

File: packages/clay-alert/stories/index.tsx [hunk 1]
```diff
@@ -95,5 +96,20 @@ storiesOf('Components|ClayAlert', module)
...
+		>
+			{'This is an alert with a button!'}
+
+			<ClayAlert.Footer>
+				<ClayButton.Group>
+					<ClayButton alert>{'View'}</ClayButton>
+				</ClayButton.Group>
+			</ClayAlert.Footer>
+		</ClayAlert>
+	))
```
Comment (matuzalemsteles) [inline packages/clay-alert/stories/index.tsx]: hey @bryceosterhaus I think we need to do more things to make the component fit the markup that @pat270 said (https://github.com/liferay/clay/issues/2765#issuecomment-569127316).

**Comment by bryceosterhaus** [comment]
@matuzalemsteles updated

**Comment by bryceosterhaus** [comment]
@matuzalemsteles good catch on that markup. I was looking in the wrong place. I just updated it now, take a look.

**Comment by bryceosterhaus** [comment]
hits "close and comment" instead of "comment" 🤦‍♂️🤦‍♂️

File: packages/clay-alert/src/index.tsx [hunk 2]
```diff
@@ -161,6 +173,33 @@ const ClayAlert: React.FunctionComponent<IClayAlertProps> & {
...
+							Refresh
+						</button>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>; 
+*/
+}
```
Comment (matuzalemsteles) [inline packages/clay-alert/src/index.tsx]: I think you just forgot about it 😅

**Comment by bryceosterhaus** [comment]
@matuzalemsteles lol I totally missed that. Just updated and removed it.

**Pull request merged**